app/controllers/sre_swagger_docs_controller.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class SreSwaggerDocsController < ApplicationController
+  def swagger_ui
+    redirect_to '/api-docs/index.html'
+  end
+
+  def open_api_json
```

Comment:
I don't usually like non-restful routes but for infrastructure work such as this it makes more sense then multiple simple controllers.